lib/PuppeteerSharp/Browser.cs

```diff
@@ -160,7 +160,7 @@ public bool IsClosed
         internal TaskQueue ScreenshotTaskQueue { get; set; }
         internal Connection Connection { get; }
         internal ViewPortOptions DefaultViewport { get; }
-        internal ChromiumProcess ChromiumProcess { get; set; }
+        internal ProcessBase ProcessBase { get; set; }
```

Comment:
We could call this `internal LauncherBase Launcher { get; set; }`

Reply:
@kblok done